test/parallel/test-http-parser-memory-retention.js

```diff
@@ -25,18 +25,24 @@ server.on('request', common.mustCall((request, response) => {
 }));
 
 server.listen(common.mustCall(() => {
-  const request = http.get({ port: server.address().port });
+  const request = http.get({
+    headers: { Connection: 'close' },
```

Comment:
This is added only to speed up the test. There is no need to wait for keep-alive to expire.